integration-tests/src/test/java/org/openmrs/maven/plugins/packager/config/ValidateContentPackageMojoTest.java

```diff
@@ -0,0 +1,101 @@
+
+package org.openmrs.maven.plugins.packager.config;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.io.FileInputStream;
+import java.util.Properties;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+public class ValidateContentPackageMojoTest {
+	
+	@InjectMocks
+	private ValidateContentPackageMojo mojo = new ValidateContentPackageMojo();
+	
+	@Mock
+	private FileInputStream mockFileInputStream;
+	
+	@Mock
+	private Properties mockProperties;
+	
+	@Rule
+	public ExpectedException exceptionRule = ExpectedException.none();
+	
+	@Before
+	public void setUp() {
+		MockitoAnnotations.initMocks(this);
+	}
+	
+	@Test
+	public void executeValidContentPropertiesFileWithoutErrors() throws Exception {
+		// setup
+		String validPropertiesFile = "src/test/resources/config-test-child/valid-content.properties";
+		mojo.sourceFile = validPropertiesFile;
+		// replay
+		mojo.execute();
+		
+		// verify no exception should be thrown
+	}
+	
+	@Test
+	public void executeInvalidContentPropertiesFormatWithException() throws Exception {
+		// setup
+		String invalidPropertiesFile = "src/test/resources/config-test-child/invalid-content.properties";
+		mojo.sourceFile = invalidPropertiesFile;
+		
+		try {
+			mojo.execute();
+		}
+		catch (MojoExecutionException e) {
+			assertTrue("MojoExecutionException raised", true);
+		}
```

Comment:
Tests should fail if the exception isn't thrown. This test always passes.

```suggestion
		try {
			mojo.execute();
			fail("Expected a MojoExecutionException to be raised")
		}
		catch (MojoExecutionException e) {
		}
```

That said, I think for this case we can just use `@Test(expected = MojoExectutionException.class)` for the same effect.

Reply:
You are correct!  Instead of asserting exception was thrown, updated code to indicate that the test expects exception to be thrown - @Test(expected = MojoExectutionException.class)